README.md

```diff
@@ -0,0 +1,43 @@
+# Babelfish Tools
+
+[![Build Status](https://travis-ci.org/bblfsh/tools.svg?branch=master)](https://travis-ci.org/bblfsh/tools)
+[![codecov](https://codecov.io/gh/bblfsh/tools/branch/master/graph/badge.svg)](https://codecov.io/gh/bblfsh/tools)
+
+Language analysis tools on top of Babelfish
+
+## Build
+
+### With docker
+
+`make build`
+
+### Without docker
+
+`make build-internal`
```

Comment:
Yes, since this is purely a Go package, just using `go build` is ok.
The Makefile does not need to use Docker either, let's use just `go build` too (e.g. src-d/go-git. src-d/borges, src-d/proteus, src-d/go-kallax).